frontend/src/utilities/useWatchNotebooksForUsers.tsx

```diff
@@ -23,7 +23,7 @@ const useWatchNotebooksForUsers = (
   const [pollInterval, setPollInterval] = React.useState(POLL_INTERVAL);
 
   const getNotebooks = React.useCallback(
-    (usernameList: string[]) => {
+    (usernameList: string[] | undefined) => {
       if (!usernameList || usernameList.length === 0) {
```

Comment:
A value is always passed in to this function.
```suggestion
    (usernameList: string[]) => {
      if (usernameList.length === 0) {
```